src/hooks/useGlobalAppState.ts

```diff
@@ -1,116 +1,51 @@
-import Minilog from 'cozy-minilog'
-import { NavigationContainerRef } from '@react-navigation/native'
-import { Route } from '@react-navigation/routers'
-import { useEffect } from 'react'
-import {
-  AppState,
-  AppStateStatus,
-  NativeEventSubscription,
-  Platform
-} from 'react-native'
-
-import * as RootNavigation from '/libs/RootNavigation'
-import { getData, StorageKeys, storeData } from '/libs/localStore/storage'
-import { routes } from '/constants/routes'
-import {
-  showSplashScreen,
-  hideSplashScreen
-} from '/app/theme/SplashScreenService'
-import { determineSecurityFlow } from '/app/domain/authorization/services/SecurityService'
+import { useNavigationState } from '@react-navigation/native'
```

Comment:
this file now "orchestrates" services according to app state. It should be renamed for sure